frontend/src/components/RightCloseButton.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { X } from "react-feather";
+
+export function RightCloseButton({ onClick }: { onClick: () => void }) {
+  return (
+    <div className="relative h-8">
+      <div className="w-10 h-10 p-2 rounded-lg justify-center items-center gap-2 inline-flex">
+        <button
+          className="absolute top-0 right-0 w-8 h-8 flex justify-center items-center rounded bg-primary/0 hover:bg-primary/5"
+          onClick={onClick}
+        >
+          <X className="h-6 w-6 text-primary" />
+        </button>
+      </div>
+    </div>
+  );
+}
```

Comment:
Kan denne komponenten forenkles litt? 😄 Ser at det brukes absolute for plassering, men vi kan kanskje basere oss på hvordan knappen er plassert i modalen? Og dermed bruke paddingen i modalen til å plassere knappen riktig. Her er et forslag som i tillegg fjerner litt tailwind som ikke har noe effekt her: 

```suggestion
export function RightCloseButton({ onClick }: { onClick: () => void }) {
  return (
    <button
      className="p-2 flex justify-center items-center rounded-lg hover:bg-primary/5 "
      onClick={onClick}
    >
      <X className="h-6 w-6 text-primary" />
    </button>
  );
}
```

Reply:
Supert! Herlig! 👏  Legger de til nå :)